tests/helpers.bash

```diff
@@ -410,6 +410,24 @@ function is_rootless() {
     [ "$(id -u)" -ne 0 ]
 }
 
+#################################
+#  skip_if_rootless_environment # `mount` or its variant needs unshare
+#################################
+function skip_if_rootless_environment() {
+    if is_rootless; then
+        skip "${1:-test is being invoked from rootless environment and might need unshare}"
+    fi
+}
+
+#################################
+#  skip_if_rootless_unshare     # `some` of the tests can be only triggered in rootfull
+#################################
+function skip_if_rootless_unshare() {
+    if [ $_CONTAINERS_ROOTLESS_UID -ne 0 ]; then
```

Comment:
This has to be double `[[`, not single. Single `[` causes an error, it does not do what you think it does.

Also, I have no idea why this function even exists, since AFAICT it will never trigger.

Reply:
@edsantiago I have removed `unshare` part to keep things simple for rootless testing.